spec/dummy/db/schema.rb

```diff
@@ -11,11 +11,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141202081811) do
+ActiveRecord::Schema.define(version: 20150112083118) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "atmosphere_action_logs", force: true do |t|
+    t.string   "message"
+    t.string   "log_level"
+    t.integer  "action_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  add_index "atmosphere_action_logs", ["action_id"], name: "index_atmosphere_action_logs_on_action_id", using: :btree
+
+  create_table "atmosphere_actions", force: true do |t|
+    t.string   "action_type"
+    t.integer  "appliance_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  add_index "atmosphere_actions", ["appliance_id"], name: "index_atmosphere_actions_on_appliance_id", using: :btree
```

Comment:
Line is too long. [115/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.